com.Genymobile.scrcpy.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.Genymobile.scrcpy</id>
+  
+  <name>scrcpy</name>
+  <summary>This application provides display and control of Android devices connected on USB (or over TCP/IP).</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+  
+  <description>
+    <p>
+      This application provides display and control of Android devices connected on USB (or over TCP/IP). It does not require any root access. It works on GNU/Linux, Windows and macOS.
+    </p>
+    <p>
+      It focuses on:     lightness (native, displays only the device screen)     performance (30~60fps)     quality (1920×1080 or above)     low latency (35~70ms)     low startup time (~1 second to display the first image)     non-intrusiveness (nothing is left installed on the device)
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">com.Genymobile.scrcpy.desktop</launchable>
```

Comment:
Is the com.Genymobile.scrcpy.desktop file included in the upstream archive?

Reply:
It is not, see [my comment](https://github.com/flathub/flathub/pull/1551#issuecomment-640092078) below.